orderapi/src/main/java/fi/hel/verkkokauppa/order/api/data/accounting/RefundItemAccountingDto.java

```diff
@@ -0,0 +1,168 @@
+package fi.hel.verkkokauppa.order.api.data.accounting;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Objects;
+
+/**
+ * Yritys
```

Comment:
Could these be moved just before variable?

Reply:
done